### PR TITLE
refactor: #1227 Changed pq to pgx in way and limit services, way, day…

### PR DESCRIPTION
**Pull request by Moon1it**
…Report, wayCollection, favoriteUser, favoriteUserWay, jobDone, jobDoneJobTag, jobTag, fromUserMentoringRequest, metric, and plan controllers.

https://github.com/tritonJS826/masters-way/issues/1227

### CODE REVIEW

File: mw-server/controllers/way.controller.go [hunk 1]
```diff
@@ -180,12 +178,13 @@ func (cc *WayController) UpdateWay(ctx *gin.Context) {
 func (cc *WayController) GetWayById(ctx *gin.Context) {
 	wayUuidRaw := ctx.Param("wayId")
 	wayUuid := uuid.MustParse(wayUuidRaw)
+	// wayPgUUID := pgtype.UUID{Bytes: uuid.MustParse(wayUUIDRaw), Valid: true}
```
Comment (tritonJS826) [inline mw-server/controllers/way.controller.go:181]:
```suggestion
```

File: mw-server/db/migration/000001_init_schema.down.sql [hunk 2]
```diff
@@ -27,9 +27,7 @@ DROP INDEX IF EXISTS way_uuid_key;
 DROP TABLE IF EXISTS ways;
 DROP TABLE IF EXISTS favorite_users;
 DROP INDEX IF EXISTS users_email_key;
-DROP TABLE IF EXISTS profile_settings;
```
Comment (tritonJS826) [inline mw-server/db/migration/000001_init_schema.down.sql:30]: why should we remove it?

File: mw-server/db/migration/000001_init_schema.up.sql [hunk 3]
```diff
@@ -265,7 +265,7 @@ FOR EACH ROW
 EXECUTE FUNCTION check_max_likes_to_user();
 
 -- максимальное число лайков которое пользователь может получить от других пользователей
-CREATE OR REPLACE FUNCTION check_max_likes_from_user() RETURNS TRIGGER AS $$
+CREATE FUNCTION check_max_likes_from_user() RETURNS TRIGGER AS $$
```
Comment (tritonJS826) [inline mw-server/db/migration/000001_init_schema.up.sql:268]: why should we remove OR REPLACE

**Review by tritonJS826 — APPROVED** [review]

File: mw-server/db/migration/000001_init_schema.down.sql (hunk 2 above)
Comment (Moon1it) [reply mw-server/db/migration/000001_init_schema.down.sql:30]: Done

File: mw-server/db/migration/000001_init_schema.up.sql (hunk 3 above)
Comment (Moon1it) [reply mw-server/db/migration/000001_init_schema.up.sql:268]: Done

File: mw-server/controllers/way.controller.go (hunk 1 above)
Comment (Moon1it) [reply mw-server/controllers/way.controller.go]: Done

File: mw-server/controllers/wayCollection.controller.go [hunk 4]
```diff
@@ -100,13 +101,14 @@ func (cc *WayCollectionController) UpdateWayCollection(ctx *gin.Context) {
 	}
 
 	now := time.Now()
-	args := &db.UpdateWayCollectionParams{
-		Uuid:      uuid.MustParse(wayCollectionId),
-		Name:      sql.NullString{String: payload.Name, Valid: payload.Name != ""},
-		UpdatedAt: sql.NullTime{Time: now, Valid: true},
+	// FIXME: If payload.Name is empty, we should not perform an update in the database
```
Comment (tritonJS826) [inline mw-server/controllers/wayCollection.controller.go:104]:
```suggestion
	// TODO: If payload.Name is empty, we should not perform an update in the database
```

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**